src/tpu.rs

```diff
@@ -381,6 +326,19 @@ impl Tpu {
         );
         Ok(())
     }
+    fn apply_entries_state(obj: &Tpu, entries: Vec<Entry>) -> Result<()> {
```

Comment:
This should be in `accountant.rs`. It has no real dependency on TPU or AccountantStage. Can you move it and update `bin/testnode.rs` to use it? Put it after `process_verified_events`.

Reply:
sure